src/ui/public/error_auto_create_index/error_auto_create_index.js

```diff
@@ -20,28 +20,20 @@
 import { get } from 'lodash';
 
 import uiRoutes from '../routes';
-import { KbnUrlProvider } from '../url';
 
 import './error_auto_create_index.less';
 import template from './error_auto_create_index.html';
 
 uiRoutes
   .when('/error/action.auto_create_index', { template });
 
-export function ErrorAutoCreateIndexProvider(Private, Promise) {
-  const kbnUrl = Private(KbnUrlProvider);
-
-  return new (class ErrorAutoCreateIndex {
-    test(error) {
-      return (
-        error.statusCode === 503 &&
-        get(error, 'body.code') === 'ES_AUTO_CREATE_INDEX_ERROR'
-      );
-    }
+export function isAutoCreateIndexError(error) {
+  return (
+    get(error, 'res.status') === 503 &&
```

Comment:
Was the original code incorrect, in which we were comparing `error.statusCode`?

Reply:
The original code used the error produced in saved_object_client _request method. But with kfetch, that is already accomplished. Just needed to update to handle the error thrown by kfetch.